scripts/port_status.py

```diff
@@ -5,12 +5,13 @@
 import networkx as nx
 import subprocess
 from urllib.request import urlopen
-
+from sys import argv
 from pathlib import Path
 
 import_re = re.compile(r"^import ([^ ]*)")
 synchronized_re = re.compile(r".*SYNCHRONIZED WITH MATHLIB4.*")
 hash_re = re.compile(r"[0-9a-f]*")
+output_git_command = 1 < len(argv)
```

Comment:
How about we avoid this kludge by simply always outputting the git command? Seems more useful, and more likely to result in people taking useful action. :-)